index.js

```diff
@@ -56,6 +60,10 @@ class K8sExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
+        const cpuConfig = hoek.reach(config, 'annotations', { default: {} })[CPU_RESOURCE];
+        const ramConfig = hoek.reach(config, 'annotations', { default: {} })[RAM_RESOURCE];
+        const CPU = (cpuConfig === 'HIGH') ? 6 : 2;
```

Comment:
CPU config is kinda different for kubernetes, can you double check?
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/

Reply:
The integer format of CPU config has been deprecated in `Kubernetes v1.8`.
So, `millicpu` format seems to be prefered.I'll fix it. Thanks!